packages/twenty-server/src/modules/reports/standard-objects/analytics-query-filter.workspace-entity.ts

```diff
@@ -0,0 +1,65 @@
+import { Relation } from 'typeorm';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import { ANALYTICS_QUERY_FILTER_STANDARD_FIELD_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids';
+import { STANDARD_OBJECT_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-object-ids';
+import { WorkspaceEntity } from 'src/engine/twenty-orm/decorators/workspace-entity.decorator';
+import { WorkspaceField } from 'src/engine/twenty-orm/decorators/workspace-field.decorator';
+import { WorkspaceIsNullable } from 'src/engine/twenty-orm/decorators/workspace-is-nullable.decorator';
+import { BaseWorkspaceEntity } from 'src/engine/twenty-orm/base.workspace-entity';
+import { RelationMetadataType } from 'src/engine/metadata-modules/relation-metadata/relation-metadata.entity';
+import { WorkspaceRelation } from 'src/engine/twenty-orm/decorators/workspace-relation.decorator';
+import { AnalyticsQueryWorkspaceEntity } from 'src/modules/reports/standard-objects/analytics-query.workspace-entity';
+import { WorkspaceJoinColumn } from 'src/engine/twenty-orm/decorators/workspace-join-column.decorator';
+
+@WorkspaceEntity({
+  standardId: STANDARD_OBJECT_IDS.analyticsQueryFilter,
+  namePlural: 'analyticsQueryFilters',
+  labelSingular: 'Analytics query filter',
+  labelPlural: 'Analytics query filters',
+  description: 'A filter for an analytics query',
+  icon: 'IconFilter',
+})
+export class AnalyticsQueryFilterWorkspaceEntity extends BaseWorkspaceEntity {
```

Comment:
If our module is called DataExplorer we can call this DataExplorerQuery? I feel like Analytics can be a bit confusing (e.g. internal analytics for the product)